conanfile.py

```diff
@@ -9,7 +9,7 @@
 
 class HomestoreConan(ConanFile):
     name = "homestore"
-    version = "7.0.4"
+    version = "8.0.1"
```

Comment:
This seems like a compatible changes,  existing pdev will still be loaded based on the pdev_info. 
Can we double comfirm it wont impacting exiting data?